websites/M/MUBI/metadata.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Not too sure why CI didn't pick up on this, but this hasn't been formatted correctly. Fix this by running lint

Reply:
ok

Reply:
Wasn't fixed, but resolved. Great!